### PR TITLE
Factories: the trilogy, Part III

### PR DESCRIPTION
**Pull request by jessieay**
* Closes https://github.com/18F/micropurchase/issues/1066 ...for now

### CODE REVIEW

File: spec/view_models/admin/user_auction_view_model_spec.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@
   describe '#user_bid_count' do
     it 'should return the total number of bids the user has made' do
       user2 = create(:user)
-      auction = create(:auction, :with_bidders, bidder_ids: [user.id, user2.id, user.id, user2.id, user.id])
+      auction = create(:auction, bidders: [user, user2, user, user2, user])
```
Comment (harrisj) [inline spec/view_models/admin/user_auction_view_model_spec.rb]: One thing I would love is the ability to pass it something like `nil` or `:new` and just let the factory create the other bidders I don't care about, but I might be overthinking

**Comment by harrisj** [comment]
This looks great. Just needs a rebase

**Comment by adelevie** [comment]
LGTM. Needs rebase.

**Comment by jessieay** [comment]
@adelevie  @harrisj rebased!

File: spec/view_models/admin/user_auction_view_model_spec.rb (hunk 1 above)
Comment (jessieay) [inline spec/view_models/admin/user_auction_view_model_spec.rb]: the `with_bids` trait creates bids with bidders -- but it sounds like you're talking about something else?

**Pull request merged**